gcc/rust/rust-gcc.cc

```diff
@@ -2057,7 +2074,7 @@ temporary_variable (tree fndecl, tree bind_tree, tree type_tree, tree init_tree,
       || error_operand_p (fndecl))
     {
       *pstatement = error_mark_node;
-      return Bvariable::error_variable ();
+      return LocalVariable::error_variable ();
```

Comment:
Not a fan of error state hidden within a single class. Couldn't we return an `Expected<LocalVariable>` ?